main.cpp

```diff
@@ -355,14 +441,25 @@ struct Turntable
     float beltSpeed = 25.4f;
     int outputs = 4;
     float needleWeight = 0.0043f;
+    Turntable();
     void playRecord();
     bool choosePlaybackSpeed(bool thirtThree);
     void antiSkipAdjust();
 };
 
+Turntable::Turntable()
+{
+    brand = "";
+    modelNum = 0;
+    beltSpeed = 0;
+    outputs = 0;
+    needleWeight = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise